src/Symfony/Component/Routing/Requirement/Requirement.php

```diff
@@ -23,7 +23,7 @@ enum Requirement
     public const POSITIVE_INT = '[1-9][0-9]*';
     public const UID_BASE32 = '[0-9A-HJKMNP-TV-Z]{26}';
     public const UID_BASE58 = '[1-9A-HJ-NP-Za-km-z]{22}';
-    public const UID_RFC4122 = '[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12}';
+    public const UID_RFC4122 = '[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12}'; // RFC 9562 obsoleted RFC 4122 but the format is the same
```

Comment:
@alexandre-daubois I guess we can add a new constant here as well?!

Reply:
Definitely! Good idea, here you go #58336